android/library/build.gradle

```diff
@@ -18,6 +18,6 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.polidea.rxandroidble2:rxandroidble:1.10.1"
+    implementation "com.polidea.rxandroidble:rxandroidble:1.7.0"
```

Comment:
Darek's advice was not to use rxjava2 here because of possible unhandled errors that can occur. These errors can be thrown when you merge a couple of streams and more than one of them emits error at the same time. Probably it is going to be handled on RxAndroidBle library side in the future, but for now it is much more safe to use rxjava1 which does not have such problems.

Reply:
Moreover it would take more time to adjust our RN bridge codebase to use rxjava2